index.js

```diff
@@ -1,18 +1,44 @@
 /***** Deliverable 1 *****/
-const header = document.querySelector("#header")
+const header = document.querySelector('h1#header')
 console.log("Here's your header:", header)
 
 
 /***** Deliverable 2 *****/
-
+const headerColor = document.querySelector('h1#header')
```

Comment:
no need to redefine this variable! You can just use "header" which you defined on line 2!